task_1_1.py

```diff
@@ -0,0 +1,22 @@
+"""
+Variant 11
+The coefficients of a system of linear equations are given as a rectangular matrix.
+With the help of permissible transformations to bring the system to a triangular form.
+Find the number of rows whose arithmetic mean is less than the given value.
+"""
+CHECK_MEAN = 5
+
+a = [[2, 5, 2, -38], [3, -2, 4, 17], [-6, 1, -7, -12]]
+column = len(a[0])
+
+for i in range(len(a)):
+    for kj in range(i):
+        a[i] = [a[kj][j] * a[i][kj] - a[i][j] for j in range(column)]
+    a[i] = [element / a[i][i] for element in a[i]]
```

Comment:
I think it cannot be rewritten to comprehensions because calculation for any line depends on calculations made for previous lines. and comprehension don't provide you ability to use it.